sw/host/opentitanlib/src/proxy/protocol.rs

```diff
@@ -17,6 +17,7 @@ pub enum Message {
 
 #[derive(Serialize, Deserialize)]
```

Comment:
To discuss in the next OpenTitanTool meeting:  Would the typetag crate be a better architectural choice than creating sum types with all possible requests and responses.